examples/arduino/z_get.ino

```diff
@@ -45,7 +45,7 @@ void reply_dropper(void *ctx) {
 void reply_handler(z_owned_reply_t *oreply, void *ctx) {
     (void)(ctx);
     if (z_reply_is_ok(oreply)) {
-        z_sample_t sample = z_reply_ok(oreply);
+        z_loaned_sample_t sample = z_reply_ok(oreply);
```

Comment:
Please revert `z_reply_ok` result type renaming `z_sample_t` / `z_loaned_sample_t` 
(`z_reply_ok` in the new API will return `const z_loaned_sample_t *`, I already did it and this renaming will only introduce additional conflicts.)